package.json

```diff
@@ -12,14 +12,15 @@
   },
   "scripts": {
     "start": "node main.js",
-    "test": "IP=0.0.0.0 PORT=3030 mocha --timeout 5000 tests/*_test.js"
+    "test": "mocha --timeout 5000 tests/*_test.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/fabric8-hdd/supervisor"
   },
   "author": "Anmol Babu <anmolbudugutta@gmail.com>",
-  "license": "Apache License 2.0",
```

Comment:
This was a generated value.
Please tell me what was the reason for this change?

Reply:
I was recieving npm warn console logs while running the scripts.

I referred to this document : https://gist.github.com/robertkowalski/7620849

and changed it to the string with value 133.

Reply:
Ok